File: poh20/cpp/2.cpp
//usr/bin/env true; tmpfile=$(mktemp); g++ -O2 -std=gnu++11 -xc++ -o $tmpfile $0 && $tmpfile "$@"; rm $tmpfile; exit
#import<iostream>
#import<vector>
#import<string>
#import<numeric>
using namespace std;
int main(){int n;cin>>n;vector<string>v(n,"BINGO!\n");cout<<accumulate(v.begin(),v.end(),string());}

File: poh20/rust/2.rs
//usr/bin/env true; tmpfile=$(mktemp); rustc -o $tmpfile $0 && $tmpfile "$@"; rm $tmpfile; exit
fn l()->String{let mut x=String::new();std::io::stdin().read_line(&mut x).ok();return x;}
fn main(){print!("{}","BINGO!\n".repeat(l().trim().parse().unwrap()));}
